packet_analysis/sql/dao/sqlObject.py

```diff
@@ -0,0 +1,19 @@
+import abc
+
+
+class sqlObject(abc.ABC):
+    """
+    Abstract class for data access objects. Packets are separated based on most specific layer.
+    """
+
+
+    @abc.abstractmethod
+    def csv(self) -> list:
+        """
+        Creates a list of the arguments required for an INSERT statement
+        :return: list
+        """
+        raise NotImplementedError
+
+# See scapy layers package to understand what fields can be pulled for each protocol:
+# https://scapy.readthedocs.io/en/latest/api/scapy.layers.html
```

Comment:
Is this necessary to include? If so it shouldn't be at the end of the file and should be placed where it's most relevant within the class.